src/commonMain/kotlin/exchange.dydx.abacus/validator/trade/TradeTriggerPriceValidator.kt

```diff
@@ -234,12 +236,13 @@ internal class TradeTriggerPriceValidator(
     private fun triggerToLiquidationError(
         triggerToLiquidation: RelativeToPrice,
         triggerLiquidation: Double,
+        tickSize: String,
     ): Map<String, Any> {
         val fields = listOf("price.triggerPrice")
         val action = "APP.TRADE.MODIFY_TRIGGER_PRICE"
         // Localizations uses TRIGGER_PRICE_LIMIT as paramater name
         val params =
-            mapOf("TRIGGER_PRICE_LIMIT" to mapOf("value" to triggerLiquidation, "format" to "price"))
```

Comment:
got rid of formatting to price because I don't think this is actually great for internationalization - the $ sign can go before or after the number depending on the language, and I don't think the formatter actually takes into account language? (maybe I'm wrong - but it feels like we should punt this responsibility to our internationalization team)

Reply:
rip actually added it back because tickSize doesn't work unless the format is "price". Will go back to localization repo and remove the extra `$`